holidayextras/brands
# Adding XL modal with max size for globals

#### What does this PR do? (please provide any background)
There is a need for a XL size modal, while at the moment this is just needed for Paultons and Peppa Pig, its likely we will need this for other brands in the near future. In the distant future i think all XL modals should be replaced by a page.

#### How can this be tested?
Link local pattern library to this brand...
`cd pattern-library && npm link ../brands`
Go to pattern library, open 
http://localhost:8687/components/modal/index.html
Edit the HTML in devTools and find `modal-dialog`. Add the class of `.modal-xl` and the modal dialog box should be 90% of the screen width (max of 1200px)

#### Any tech debt?
None

#### Screenshots / Screencast
<img width="1266" alt="screen shot 2017-02-15 at 10 14 33" src="https://cloud.githubusercontent.com/assets/778942/22969768/9c5cb2b6-f367-11e6-8a0c-57dbdf089b2d.png">

#### What gif best describes how you feel about this work?
![giphy-7](https://cloud.githubusercontent.com/assets/778942/22969850/d6969e2e-f367-11e6-848d-6c33f1ea02e2.gif)



---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---
